### PR TITLE
#235: optinal BankConnection.owners()

### PR DESCRIPTION
**Pull request by ccostin93**

### CODE REVIEW

**Comment by ccostin93** [comment]
@proshin-roman Unfortunately, this is a breaking change. I had to change 
`Iterable<Owner> BC.owners` to `Optional<Iterable<Owner>> BC.owners`

File: src/main/java/org/proshin/finapi/bankconnection/FpBankConnection.java [hunk 1]
```diff
@@ -131,11 +132,15 @@ public boolean collectiveMoneyTransferSupported() {
     }
 
     @Override
-    public Iterable<Owner> owners() {
-        return new IterableJsonArray<>(
-            this.origin.getJSONArray("owners"),
-            (array, index) -> new FpOwner(array.getJSONObject(index))
-        );
+    public Optional<Iterable<Owner>> owners() {
```
Comment (proshin-roman) [inline src/main/java/org/proshin/finapi/bankconnection/FpBankConnection.java:135]: Optional of Iterable doesn't make much sense as the iterable itself can be empty. Let's maybe map `null` value to an empty Iterable?

**Review by proshin-roman — CHANGES_REQUESTED** [review]

**Review by proshin-roman — APPROVED** [review]

**Comment by ccostin93** [comment]
@dependabot squash and merge

**Comment by ccostin93** [comment]
@rultor merge

**Comment by proshin-roman** [comment]
@rultor confirm

**Pull request merged**